lib/dom-to-image.js

```diff
@@ -545,8 +552,11 @@
       return array
     }
 
-    function escapeXhtml(string) {
-      return string.replace(/%/g, '%25').replace(/#/g, '%23').replace(/\n/g, '%0A')
+    function escapeXhtml(string, escapePercentSign) {
+      if (escapePercentSign) {
```

Comment:
@qw-in I'm pretty sure that instead of passing this in as a prop, we can just update the code path that converts to PNG to escape the percent sign, and don't convert that in the path used by SVGs.